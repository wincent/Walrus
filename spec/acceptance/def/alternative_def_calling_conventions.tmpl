## This example shows some alternative ways of causing #def content to appear in the output
## (by default #def content is not automatically injected into the output)
#def block1
This is block 1. It will be injected using the \#silent directive.
#end
#def block2
This is block 2. It will be injected using the \#ruby directive.
#end
#silent block1### no need to call "accumulate" here because the block already does that.
#ruby
# no need to call "accumulate" here because the block already does that.
block2
#end