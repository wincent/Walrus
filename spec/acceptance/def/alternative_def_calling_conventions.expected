This is block 1. It will be injected using the #silent directive.

This is block 2. It will be injected using the #ruby directive.
